app/liquid_tags/golang_tag.rb

```diff
@@ -0,0 +1,29 @@
+class GolangTag < LiquidTagBase
+  PARTIAL = "liquids/golang".freeze
+  URL_REGEXP = %r{\Ahttps?://play.golang.org/p/[a-zA-Z0-9\-/]+\z}.freeze
+
+  def initialize(_tag_name, link, _parse_context)
+    super
+    stripped_link = ActionController::Base.helpers.strip_tags(link)
+    the_link = stripped_link.split(" ").first
+    @embedded_url = valid_link?(the_link)
```

Comment:
This won't be a URL, but a boolean value. This can also 
<img width="733" alt="Screen Shot 2020-08-03 at 09 11 41" src="https://user-images.githubusercontent.com/47985/89139584-5d310f00-d569-11ea-9161-53951012464d.png">
be seen by inspecting the rendered template in the spec:

```
<iframe
  title="golang editor"
  src="true"
  height="400"
  frameborder="0"
  loading="lazy"
>
</iframe>
```

Because of the incorrect `src` attribute, the `iframe` will render a 404 page:

<img width="725" alt="Screen Shot 2020-08-03 at 09 13 30" src="https://user-images.githubusercontent.com/47985/89139656-9b2e3300-d569-11ea-8252-0f0ae4fc42d7.png">

I suggest the following fix:

```suggestion
    raise "Invalid URL" valid_link?(the_link)
    @embedded_url = the_link
```